lib/tasks/cleanup.rake

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+namespace :cleanup do
+  desc "Destroy trainees based on email list"
+  task :discard_trainees, [:file_path] => :environment do |_t, args|
+    raise "You must provide a file path" unless args[:file_path]
+
+    emails = File.readlines(args[:file_path]).map(&:chomp)
+    trainees = Trainee.where(email: emails)
+    trainees.discard_all
```

Comment:
How about adding the variation which will raise an Exception if anything goes wrong? [discard_all!](https://github.com/jhawthorn/discard/blob/c52de70bca2521a193f85dcb2424afa89074bddc/lib/discard/model.rb#L61)

```suggestion
    trainees.discard_all!
```